client/src/pages/Login.tsx

```diff
@@ -1,44 +1,45 @@
 import { observer } from "mobx-react-lite";
 import { authStore } from "../stores/AuthStore";
 import { useState } from "react";
+import Input from "../components/UIElements/Input";
+import Button from "../components/UIElements/Button";
+import { Link, useNavigate } from "react-router-dom";
+import Card from "../components/UIElements/Card";
 
 const Login = observer(() => {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
```

Comment:
email, and password state are causing the whole component render.
Not very critical but its somthing you should know happening.
You can use react-dev-tools chrome extension to see the rendering

Reply:
how am I suppose to manage that without state?
would love a demo about react-dev-tools :)